### PR TITLE
provide release build

### PR DESCRIPTION
**Pull request by jinyangyang222**
fixes eclipse-xpanse/xpanse#717

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.BOT_GITHUB_TOKEN }}
+
+      - name: Set Up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: 17
+          distribution: 'temurin'
+          gpg-private-key: ${{ secrets.ORG_GPG_PRIVATE_KEY }}
```
Comment (swaroopar) [inline .github/workflows/release.yml:46]: these lines upto line 47 can be removed.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+      - name: Push POM updates with release version
+        uses: EndBug/add-and-commit@v9
+        with:
+          message: "[GitHub Action] Update release version"
+          author_name: ${{ env.BOT_USER_NAME }}
+          committer_name: ${{ env.BOT_USER_NAME }}
+          author_email: ${{ env.BOT_EMAIL_ID }}
+          committer_email: ${{ env.BOT_EMAIL_ID }}
+
+      - name: Release to Maven Central
```
Comment (swaroopar) [inline .github/workflows/release.yml:111]: we must change this name. It can be "Release new version on GitHub"

**Review by swaroopar — CHANGES_REQUESTED** [review]
comments added.

File: .github/workflows/release.yml (hunk 1 above)
Comment (jinyangyang222) [reply .github/workflows/release.yml]: fixed

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**